src/Tribe/Date_Utils.php

```diff
@@ -459,6 +459,30 @@ public static function is_all_day( $all_day_value ) {
 			);
 		}
 
+		public static function range_coincides( $range_1_start, $range_1_end, $range_2_start, $range_2_end ) {
```

Comment:
Good call to @borkweb I meant, I think we were commenting at the same time. :) IDK about the more than 2 ranges thing. Do we have a need for that?